cypress/integration/unit_tests/add_disaster_test.js

```diff
@@ -261,55 +267,52 @@ describe('Unit tests for add_disaster page', () => {
 
   it('tests list cell', () => {
     const chocolate = 'chocolate';
+    const chai = 'chai';
+    const nutmeg = 'nutmeg';
+
+    setDisasterAndLayers([{flavors: [chocolate, chai]}]);
```

Comment:
Nit: extract object with flavors property to variable and use below?

Reply:
Done.